### PR TITLE
Update wording and provide new screenshots

### PR DESCRIPTION
**Pull request by Tediore**
The onboarding page was out of date given the additions to the onboarding process. I provided some new screenshots and updated the description of the process.

I added some new screenshots to the automation page.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by Tediore** [comment]
Alright, I think I got it right this time 🙄 😆

**Comment by klaasnicolaas** [comment]
Yes it went well 👍

**Comment by Tediore** [comment]
I changed the default branch in my fork to `current`. Looks like the default is `next` when you fork it, at least on my end.

File: source/getting-started/automation.markdown [hunk 1]
```diff
@@ -6,36 +6,43 @@ redirect_from:
  - /getting-started/automation-2/
 ---
 
-When your devices are set up, it's time to put the cherry on the pie: automation. In this guide we're going to create a simple automation rule to **turn on the lights when the sun sets**.
+Once your devices are set up, it's time to put the cherry on the pie: automation. In this guide we're going to create a simple automation rule to **turn on the lights when the sun sets**.
 
-In Home Assistant, open the menu by clicking on the top-left icon and click on configuration. Now click on automation. This is the automation screen from which you can manage all the automations in Home Assistant.
+In Lovelace, click Configuration in the sidebar, then click Automation. You will now see the automation screen from which you can manage all the automations in Home Assistant.
```
Comment (frenck) [inline source/getting-started/automation.markdown]: This is not Lovelace specific.

```suggestion
In Home Assistant, click Configuration in the sidebar, then click Automation. You will now see the automation screen from which you can manage all the automations in Home Assistant.
```

File: source/getting-started/onboarding.markdown [hunk 2]
```diff
@@ -5,19 +5,43 @@ description: "Instructions to get Home Assistant configured."
...
-With Home Assistant installed, it's time for doing the initial configuration. Here you will create the owner account of Home Assistant. This account will be an administrator and will always be able to change everything. Enter a name, username, password and click on "create account".
+With Home Assistant installed, it's time to configure it. Here you will create the owner account of Home Assistant. This account will be an administrator and will always be able to change everything. Enter a name, username, password and click on "create account".
 
-Now you're brought to the main screen of Home Assistant: the states screen. The states screen will show all your devices. So let's get that screen filled up!
+<p class='img'>
+<img src='/source/getting-started/screenshots/01 - username.png' />
+Set your username and password.
+</p>
+
+Next, you can enter a name for your home and set your location and unit system. Click DETECT to find your location and set your time zone and unit system based on that location. If you'd rather not send your location, you can set these values manually.
```
Comment (frenck) [inline source/getting-started/onboarding.markdown]:
```suggestion
Next, you can enter a name for your home and set your location and unit system. Click "DETECT" to find your location and set your time zone and unit system based on that location. If you'd rather not send your location, you can set these values manually.
```

File: source/getting-started/onboarding.markdown [hunk 3]
```diff
@@ -5,19 +5,43 @@ description: "Instructions to get Home Assistant configured."
...
+</p>
+
+Once you are done, click Next. In this screen, Home Assistant will show any devices that it has discovered on your network. Don't be alarmed if you see fewer items than what is shown below; you can always manually add devices later.
+
+<p class='img'>
+<img src='/source/getting-started/screenshots/04 - devices.png' />
+Discovery of devices on your network.
+</p>
+
+Finally, click Finish. Now you're brought to the main screen of Home Assistant called Lovelace. This screen will show all of your devices. So let's get that screen filled up!
```
Comment (frenck) [inline source/getting-started/onboarding.markdown]: The screen is not called Lovelace. Lovelace is only a part of the full frontend.

File: source/getting-started/onboarding.markdown [hunk 4]
```diff
@@ -5,19 +5,43 @@ description: "Instructions to get Home Assistant configured."
...
 <p class='img'>
-<img src='/images/getting-started/integrations.png' />
+<img src='/source/getting-started/screenshots/07 - integrations.png' />
 The integrations page in the configurations panel shows you all your configured integrations.
 </p>
 
 When each integration is done setting up, it will ask you to put the new devices in areas. Areas allow you to organize all the devices in your home.
 
-When you're done, navigate back to the states panel and voila, your devices are ready for you to control.
+When you're done, navigate back to Lovelace and voila, your devices are ready for you to control.
```
Comment (frenck) [inline source/getting-started/onboarding.markdown]: We should avoid Lovelace here. It is just the dashboard or Home Assistant frontend.
Lovelace is still optional + not the whole interface is Lovelace.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by Tediore** [comment]
Thanks Frenck! Sorry for the misunderstanding. I reverted my changes on the last two comments--thoughts?

**Comment by frenck** [comment]
Well, we currently have both the old state's screen and Lovelace for displaying information about your system. Besides that, there is the other things our frontend provides.

Maybe just call it the "Home Assistant web interface"? It covers it all right? To be clear on the subject, I'm not quite sure myself... I'm just opening up the discussion on this.

**Comment by Tediore** [comment]
Right. I changed it to Lovelace since I was editing the onboarding page and Lovelace is the default. But yes, "Home Assistant web interface" would be a good general term!

I'll change that soon.

**Comment by Tediore** [comment]
Alright, how about now? 😉

**Comment by frenck** [comment]
Great progress! Now the thing left, is to fix the images... Please check the build preview 😉

**Comment by Tediore** [comment]
It looks ok to me. [Am I looking in the wrong spot](https://github.com/home-assistant/home-assistant.io/blob/fdf733e2a813da258ab84ec8f790a46919a94b4b/source/getting-started/onboarding.markdown)? (The answer I'm sure is yes, otherwise you wouldn't have sent your message 😆 )

**Comment by frenck** [comment]
Yes, you are. The GitHub preview is an approximation, however, it is not accurate.
If you click on the Netlify build, you'll go to an actual build preview of the whole website, based on this PR.
You'll then notice the broken images.

**Comment by Tediore** [comment]
Ah, thank you. I added those images to my fork and they're included in this PR. Are they not showing up because the PR hasn't been merged yet? If not, how should I resolve this?

Forgive my ignorance--still learning about Github.

**Comment by frenck** [comment]
The paths in this repository do not match 1-1 with the generated result.

I would recommend:

- Move the images to `/source/images/getting-started/`
- Rename the images to have filenames that do not contain spaces or capital letters and are not numbered (order might change in the future) (e.g.,  `12 - new trigger.png` -> `new-trigger.png`)
- Use image refs: `/images/getting-started/new-trigger.png`

(the `/source/` folder to does not show in the final result)

While you are in `/image/getting-started/`, that folder contains the "old" images, it would be nice if you could clean those up as well.

**Comment by Tediore** [comment]
Will do! Sorry for all the trouble with this...

**Comment by frenck** [comment]
Don't worry @Tediore! You are doing fine!

**Comment by Tediore** [comment]
Ok. Moved the new screenshots to the requested directory and removed the old screenshots. Just waiting on the netlify build to be ready to confirm the image paths are correct.

Sorry it took me so long to do this. Thoughts?

**Comment by Tediore** [comment]
Also, I've now included the changes to `configuration.markdown` in this PR (from the other PR I closed).

**Review by frenck — APPROVED** [review]
Awesome job @Tediore! (And nicely recovered the PR as well!)  👍 

Thank you!

**Pull request merged**